apps/antalmanac/src/lib/download.ts

```diff
@@ -153,7 +155,7 @@ export function getFirstClass(
  * // examEnd = [2019, 6, 7, 12, 30]
  * ```
  */
-export function getExamTime(exam: WebsocSectionFinalExam, year: number): [DateTimeArray, DateTimeArray] | [] {
+export function getExamTime(exam: CourseEvent['finalExam'], year: number): [DateTimeArray, DateTimeArray] | [] {
```

Comment:
Could you go back into the `CourseCalendarEvent` file and extract the type of `CourseEvent['finalExam']` into it's own type? It's convenient to have direct access to the types you need instead of accessing properties of a parent type.

```suggestion
export function getExamTime(exam: FinalExam, year: number): [DateTimeArray, DateTimeArray] | [] {
```